src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -186,6 +187,20 @@ def _get_payment_info(self, url, payload):
             "configuration": self._get_configuration(),
         }
 
+    def tokenize_card(self, order, billing_address):
```

Comment:
Can we pass a `paymentMethodToken` to tokenize a card ?

As from frontend, at imprint step, the user will be able use a registered credit card but if this one has been registered from a regular payment, the `initialIssuerTransactionIdentifier` will be null...

So IMO, we should be able to tokenize a registered credit card.

Maybe this topic can be addressed in another PR.

Reply:
I'm not sure to understand how a registered card can be tokenize ? A tokenize card is a registered card.

Reply:
Sorry I shuffle two concepts : credit card token and credit card initial issuer transaction identifier.

When we tokenize a card, we get those two informations but in case of a regular payment where the user ask to save its card, the initial issuer transaction identifier will be None. Then I believed that without this identifier we were not able to do a zero click payment.

But according to the documentation that @kernicPanel shows me (https://github.com/openfun/joanie/pull/766#discussion_r1601543401), without this identifier we are able to do a zero click payment but we may have a soft decline.

My question was, if a user has save a card during a regular payment (so without identifier), can we ask to tokenize this card to fulfill the `initial_issuer_transaction_identifier` field

Reply:
Ok so after the discussion of this morning, @kernicPanel check and on our test environment, the initialIssuerTransactionIdentifier is always fulfill. So there is no problem.